pandas_datareader/tests/test_fred.py

```diff
@@ -23,11 +23,13 @@ def test_fred(self):
 
         received = web.DataReader("GDP", "fred", start, end)['GDP'].tail(1)[0]
 
-        # < 7/30/14 16535 was returned
+        # < 2014-07-30 16535 was returned
         # self.assertEqual(int(received), 16535)
-        # < 8/20/15 16502 was returned
+        # < 2015-08-20 16502 was returned
         # self.assertEqual(int(received), 16502)
-        self.assertEqual(int(received), 16440)
+        # < 2016-08-01 16440 was returned
+        # self.assertEqual(int(received), 16440)
+        self.assertEqual(int(received), 16475)
```

Comment:
ideally we should compare expected df....though have to maintain the test when previous indicator has been updated :weary:

I think it may better to compare ``index`` at least to check ``start`` and ``end`` works as expected.

```
web.DataReader("GDP", "fred", start, end)
#                 GDP
# DATE
# 2010-01-01  14681.1
# 2010-04-01  14888.6
# 2010-07-01  15057.7
# 2010-10-01  15230.2
# 2011-01-01  15238.4
# 2011-04-01  15460.9
# 2011-07-01  15587.1
# 2011-10-01  15785.3
# 2012-01-01  15973.9
# 2012-04-01  16121.9
# 2012-07-01  16227.9
# 2012-10-01  16297.3
# 2013-01-01  16475.4
```